src/Shared-Src/Native/util.h

```diff
@@ -131,6 +137,90 @@ namespace shared
         }
     };
 
+
+
+    const ULONG kEnumeratorMax = 256;

```

Comment:
Why implementing an Enumerator since you get only the first one ?
Maybe a class that would get the info you want and cleanup  after (destructor)

Reply:
The Enumerator class is used in the tracer to handle those enums from the unmanaged api easily, due we plan to merge both repos and share those utility libraries, seems reasonable to use it.

Reply:
thx for the explanation